src/main/cpp/src/get_json_object.hpp

```diff
@@ -53,11 +53,18 @@ std::unique_ptr<cudf::column> get_json_object(
  * This function processes all the JSON paths in parallel, which may be faster than calling
  * to `get_json_object` on the individual JSON paths. However, it may consume much more GPU
  * memory, proportional to the number of JSON paths.
+ * @param input the input string column to parse JSON from
+ * @param json_paths the path operations to read extract
+ * @param memory_budget_bytes a memory budget for temporary memory usage if > 0
+ * @param parallel_override if > 0 then teh number of paths to process in parallel
+ *  this will cause the memory_budget_bytes to be ignored.
```

Comment:
```suggestion
 * @param parallel_override if > 0 then specify the number of paths to process in parallel 
 *        (this will cause the `memory_budget_bytes` to be ignored)
```